lambda_function.py

```diff
@@ -1,8 +1,13 @@
+import os
 import json
-
+import urllib3
 
 def handler(event, context):
     decoder = json.decoder.JSONDecoder()
     message = decoder.decode(event['Records'][0]['Sns']['Message'])
     print("Consignment reference:", message["consignment-reference"])
+    http = urllib3.PoolManager()
+    metadata_file = http.request('GET', message["s3-folder-url"])
+    metadata = decoder.decode(metadata_file.data.decode('utf-8'))
+    print(metadata["bagit-info"]["Internal-Sender-Identifier"])
```

Comment:
Is this debug or a necessary part of the code?

Reply:
I think you might be looking at an old diff, that's been removed now

Reply:
🤦